dns_sd_windows.c

```diff
@@ -0,0 +1,262 @@
+// Based on the example provided here: https://github.com/mjansson/mdns
```

Comment:
We should say which specific file, who was the author, and what license it has (the public domain), to make sure that we can include it in a LGPL project.